helper/test/table.c

```diff
@@ -29,9 +29,9 @@ int main(int argc ODP_UNUSED, char *argv[] ODP_UNUSED)
 	odph_table_t tmp_tbl;
 	odph_table_ops_t *test_ops;
 	char tmp[32];
-	char ip_addr1[] = "12345678";
-	char ip_addr2[] = "11223344";
-	char ip_addr3[] = "55667788";
+	char key1[] = { 0, 10, 11, 12, 13, };
```

Comment:
I am not sure the ARP comment really adds much value and would not mind it being removed. Then you could also rename macaddr to value.